sources/packages/backend/apps/api/src/route-controllers/application/application.controller.service.ts

```diff
@@ -218,6 +219,26 @@ export class ApplicationControllerService {
     return details;
   }
 
+  /**
+   * Transformation util for application header information for AEST DTO.
+   * @param applicationDetail
+   * @returns application header DTO
+   */
+  async transformToApplicationHeaderForAESTDTO(
```

Comment:
Please remove this code as stated in https://github.com/bcgov/SIMS/pull/3734/files#r1777448897

Reply:
Removed.